HedgeLib/src/archives/hl_pacx.c

```diff
@@ -1855,9 +1947,13 @@ static HlResult hlINPACxV4DecompressToBlobs(HlBlob* HL_RESTRICT pac,
     if (rootHeader->splitCount)
     {
         HlPACxV3SplitTable* splitTable = hlPACxV3GetSplitTable(rootHeader);
+
         HlPACxV4SplitEntry* splitEntries = (HlPACxV4SplitEntry*)
             hlOff64Get(&splitTable->splitEntries);
 
+        HlPACxV403SplitEntry* splitEntriesV403 = (HlPACxV403SplitEntry*)
```

Comment:
Maybe just change the type of splitEntries to void* and then cast it to the type needed later on?

Idk there's nothing really wrong with this but I think it's simpler to just have one pointer rather than two that point to the same thing in this case. Also makes it easier if they add another PACx V4 revision in the future.